src/lamplib/src/genny/tasks/preprocess.py

```diff
@@ -279,6 +282,44 @@ def _replace_param(self, input):
         else:
             return self._recursive_parse(defaultVal)
 
+    def _replace_numexpr(self, input):
+        if "Expr" not in input:
+            msg = "Invalid keys for '^NumExpr', please set 'Expr'" f" in following node: {input}"
+            raise ParseException(msg)
+
+        if type(input["Expr"]) != str:
+            msg = "Invalid value for 'Expr', which must be a string," f" in following node: {input}"
+            raise ParseException(msg)
+
+        parsedDict = {}  # Pass empty dict to avoid yaml to access context of this function
+        if "Dict" in input:
```

Comment:
Could we call this "args" or "values"? I'm not sure it's obvious how a dict would necessarily be used.

Reply:
Thanks for the feedback. Please see latest commits where this has been changed. Also added an example to HelloWorld.yml